modules/custom/openy_search/openy_google_search/openy_google_search.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  openy_google_search.overrider:
+    class: Drupal\openy_google_search\Config\GoogleSearchOverrides
```

Comment:
The problem here is that it must be in the openy_search module, not in openy_google_search. You should be able to enable both backends so that you can even create a page with both paragraphs and compare the results.

Reply:
Moved to openy_search module